i18n/en.json

```
@@ -940,6 +940,9 @@
   "admin.license.noFile": "No file uploaded",
   "admin.license.removing": "Removing License...",
   "admin.license.title": "Edition and License",
+  "admin.license.trial-request.different-data": "Or get a trial license with different data here: ",
```

Are we keeping the mattermost.com/trial link if requesting the trial license automatically fails? I'm just wondering if we'd remove it altogether.

Else, I'd tweak the text to:

```suggestion
  "admin.license.trial-request.different-data": "Or get a trial license manually at ",
```

since it may not be obvious to the admin if they should use different data.

---

Yes, on error the error text is shown below, the rest of the text is keep as is

---

Changed the text proposed.